### DISCUSSION TITLE
Fails de l'atk sombre du mage

### DISCUSSION THREAD

**Discussion by Carsebel** [Crownicles/Crownicles#2518] [Nouvelles idées]
Bonjour,
Je voulais savoir savoir si tous les ratés de l'atk sombre étaient normaux? 
Car ça fait un peu beaucoup quand sur 5 fights, la sombre ne fonctionne que bien que 1 match sur 2 .

**Comment by Ntalcme** [comment]
Salut, tu peux voir le fichier sur l'attaque sombre [ici](https://github.com/DraftBot-A-Discord-Adventure/DraftBot/blob/master/src/core/fights/actions/interfaces/players/darkAttack.ts) si tu veux mais je ne constate aucun souci donc je dirais que tout est normal oui.

**Comment by Carsebel** [comment]
et si on peut rajouter la bénédiction. 
Trop d'aléatoire dans ces attaques dernièrement.

**Reply by Ravenclaw-l** [reply]
Oui, je confirme. Pour une attaque utilisable une seule fois par combat, elle manque de fiabilité. 
Le chevalier est devenu médiocre en pvp et la béné ne l'aide pas
